ieeeuoft/makeuoft-2023
# added qr scanner code

## Overview

- adds qr code scanning to makeuoft.ca
- scanner only appears when admins login


## Unit Tests Created

- none for now


## Steps to QA

- run migrations, collectstatic and compile scss
- as a participant, take a photo of ur qrcode when you login
- login as an admin, go to qr scanner page and scan ur participant code with camera
- might need to change times in settings/__init__.py
- check for errors:
- throws an error for duplicate sign-ins
- participant who hasn't RSVP'd and isn't accepted or a user that doesn't exist
- sign in data should appear on django admin site (model called UserActivity)
- test for general user experience and fluidity
